app/poem.tsx

```diff
@@ -0,0 +1,11 @@
+import { Text, View } from "react-native";
+
+const Poem = () => {
+  return (
+    <View>
+      <Text>Poem</Text>
+    </View>
+  );
+};
+
+export default Poem;
```

Comment:
I would structurize it as `app/poem/Poem.tsx` and add `index.ts` there. I wonder if `expo-router` would create 2 separate routes then 🤔

Reply:
expo-router requires index.ts on the root anyway, so then I would need to use useEffect to redirect to Poem.tsx, is it a good idea to complicate it if we don't have any actions to do before initialization?

Reply:
I think that for poems we need a dynamic route in order to pass some ids -> https://docs.expo.dev/router/create-pages/#dynamic-routes

Reply:
And yeah, for this case it will be easier to keep this `index.tsx` file in the root to not overcomplicate things and avoid unexpected behaviours (we don't have that much practical experience with expo-router ;p)